libs/model/src/community/UpdateGroup.command.ts

```diff
@@ -90,6 +90,26 @@ export function UpdateGroup(): Command<
               transaction,
             },
           );
+
+          // update topic level interaction permissions for current group
+          await Promise.all(
```

Comment:
Same here

Reply:
this one is actually for update, lmk if you think a raw query would be fine here

Reply:
ah right - raw query would be messy here - not worth it.